syntax = "proto3";

package LAMARCK_ML;

import "LAMARCK_ML/data_util/Shape.proto";
import "LAMARCK_ML/data_util/DType.proto";

message TypeShapeProto {
    string id_name = 1;
    DTypeProto dtype_val = 2;
    ShapeProto shape_val = 3;
}